command_line_assistant/commands/chat.py

```diff
@@ -538,9 +538,45 @@ def execute(self) -> None:
 class SingleQuestionOperation(BaseChatOperation):
     """Class that holds the single question ask operation"""
 
+    def _validate_query(self):
+        """Helper function to validate query size.
```

Comment:
More appropriately, this function validates the query in general. Size won't be the only validation added to this function eventually, I'm certain.
```suggestion
        """Helper function to validate query.
```

Reply:
Good catch. Thanks!